releasenotes/notes/0.16/4953-Inverse-of-MCX-gates-generated-without-qubits-for-ancillae-d8858552eb2cd3f9.yaml

```diff
@@ -0,0 +1,29 @@
+---
+fixes:
+  - |
+    The :class:`~qiskit.circuit.libray.SXGate` class's
+    :meth:`~qiskit.circuit.library.SXGate.inverse` method had previously
+    returned an ``sx_dg`' gate with a correct
+    ``definition`` but incorrect ``to_matrix()`` method. This has been fixed
+    so that :meth:`qiskit.circuit.library.SXGate.inverse` returns
+    a :class:`~qiskit.circuit.library.SXdgGate` object and vice versa.
+  - |
+    Previously, the :class:`~qiskit.circuit.Instruction` class's
+    :meth:`~qiskit.circuit.Instruction.inverse` method would in some cases
+    (when not overriden by a subclass) return a :class:`~qiskit.circuit.Gate`
+    instance with an incorrect :meth:`~qiskit.circuit.Gate.to_matrix` method.
+    This has been fixed so that these cases with an invalid
+    :meth:`~qiskit.circuit.Gate.to_matrix` no longer occur.
+  - |
+    Previously, a :class:`~qiskit.circuit.libray.C3XGate` object created with a
+    non-zero ``angle`` argument would generate an incorrect gate when
+    inverting the gate with :meth:`~qiskit.circuit.libray.C3XGate.inverse`.
+    This issue has been fixed and the correct gate is now returned by
+    :meth:`qiskit.circuit.libray.C3XGate.inverse`.
+  - |
+    The :class:`~qiskit.circuit.MCXGate` modes have been updated to return a
+    gate of the same mode when calling
+    :meth:`~qiskit.circuit.libray.MCXGate.inverse`. This fixes an issue where
+    in some cases, transpiling a circuit containing the inverse of an
+    :class:`~qiskit.circuit.MCXVChain` gate would raise an error.
+    Fixes `#4953 <https://github.com/Qiskit/qiskit-terra/issues/4953>`__
```

Comment:
These have been backported in 0.15.2.

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/5142/commits/bdfce87f1dda045ec61dd740645aed055df041ee